### DISCUSSION TITLE
Does Zed have intellisense feature similar to VSCode `Command + i` ?

### DISCUSSION THREAD

**Discussion by mokuzuu** [zed-industries/zed#6589] [Feature Requests]
Hi guys! I just started using Zed and love it! i just wonder whether Zed has similar feature to VSCode IntelliSense (by pressing `Command + i`)?

**Comment by ivansvlv** [comment]
@mokuzuu Yes, you can use `Ctrl + Space` to show suggestions/completions.

You can find it, and re-bind it in Key Bindings: `"ctrl-space": "editor::ShowCompletions"`

**Reply by mokuzuu** [reply]
thank you @ivansvlv 🙏
